zivid_description/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(zivid_description)
+find_package(catkin REQUIRED COMPONENTS)
+catkin_package()
+include_directories(${catkin_INCLUDE_DIRS})
```

Copy-pasta @dave992 ?

---

I'd also suggest to add `install(..)`:

```suggestion
install(DIRECTORY config launch meshes urdf
  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```